package.json

```diff
@@ -66,9 +66,6 @@
     "vue-template-compiler": "^2.6.11",
     "vuetify-loader": "^1.3.0"
   },
-  "gitHooks": {
```

Comment:
This was a change from awhile back that somehow got lost in a merge. Just thought I'd put it in while I was here